README.md

```diff
@@ -40,3 +40,113 @@ Run `helm upgrade --install <release-name> <path-to-chart> --namespace <namespac
 ### Default Namespace
 
 This installer is incompatible with `default` namespace installations, install and uninstall commands must include `--namespace <target-namespace>` or the context namespace must be changed, e.g. `oc project <target-namespace>`.
+
+## GitOps/ArgoCD Configuration
+
+As part of this section you will find information about the following configuration methods:
+
+1. Configuring GitOps/ArgoCD with the configuration script after using our `ai-rhdh-installer`
+2. Configuring GitOps/ArgoCD manually after using our `ai-rhdh-installer`
+3. Configuring GitOps/ArgoCD with the configuration script for a pre-existing ArgoCD and RHDH instance
+4. Configuring GitOps/ArgoCD manually for a pre-existing ArgoCD and RHDH instance
+
+### Prerequisites
+<!---
+TODO: Once RHDH configuration is complete we should link a reference to it here.
+-->
+- In order to allow the configuration to be completed you must first have a working RHDH instance (for example you are able to login and view developer hub).
+- [yq](https://github.com/mikefarah/yq/) version 4.0+.
+
+### AI-RHDH-Installer: Script Configuration
```

Comment:
Can we move these instructions to a new MD file and link it here on the main README?

The main README seems very lengthy now

Reply:
Moved the section for GitOps/ArgoCD config to it's own file and referenced it within `README.md`